plugin/pkg/admission/priority/admission.go

```diff
@@ -81,9 +79,7 @@ func (p *Plugin) ValidateInitialization() error {
 }
 
 // InspectFeatureGates allows setting bools without taking a dep on a global variable
-func (p *Plugin) InspectFeatureGates(featureGates featuregate.FeatureGate) {
-	p.nonPreemptingPriority = featureGates.Enabled(features.NonPreemptingPriority)
-}
+func (p *Plugin) InspectFeatureGates(featureGates featuregate.FeatureGate) {}
```

Comment:
I think you can remove this, along with L61.

Reply:
thanks  Done